crates/sncast/src/lib.rs

```diff
@@ -351,26 +303,27 @@ pub fn get_account_data_from_keystore(
         get_keystore_password(KEYSTORE_PASSWORD_ENV_VAR)?.as_str(),
     )?
     .secret_scalar();
-    let private_key = format!("{private_key:#x}");
 
     let account_info: Value = read_and_parse_json_file(&path_to_account)?;
 
-    let parse_to_string =
-        |pointer: &str| -> Option<String> { get_string_value_from_json(&account_info, pointer) };
+    let parse_to_felt = |pointer: &str| -> Option<FieldElement> {
```

Comment:
Shouldn't this be part of `conversions`? 🤔

Reply:
Isn't conversion for Rust <-> Cairo? Also this part will be deleted anyway on next PR.